.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,19 @@
-<!-- Please add the Jira issue id to the Pull Request title, like this: "[HVG-123] Make the Pull Request template great again" -->
+<!-- 
+If there is one, add the ID of the Jira issue where this issue is described to the Pull Request title.
```

Comment:
I think "where this issue is described" can be remove. I think it just makes the sentence a bit more difficult to read.
a suggestion would be "If a Jira issue exists, add the issue ID to the Pull Request title"

Reply:
Sure it might be a bit lengthy 👍